shared/parcel.js

```diff
@@ -10,6 +10,16 @@ export function isParcel(asset) {
   )
 }
 
+export function getParcelSorter() {
```

Comment:
Just a thought: we could name it as `parcelSortAlgorithm` or just `sortParcel`

Reply:
"sortParcel" would be the _return value_ of this function, since that's the actual function that does the sorting. I think it's fine returning it as a lambda tho, as it is doing now.